manage-data/migrate.md

```diff
@@ -107,86 +115,95 @@ Through the {{es}} reindex API, you can connect your new {{es}} Service deployme
-    :::
+    **Do not** configure the repository as an [ECE-managed repository](/deploy-manage/tools/snapshot-and-restore/cloud-enterprise.md), which is intended for automatic snapshots of deployments. In this case, you need to add a custom repository that already contains snapshots from another cluster.
+    ::::
 
-    :::{tab-item} {{ece}}
 
-    From the Cloud UI of the **new** {{es}} cluster add the snapshot repository.
+### Step 2: Run the snapshot restore [migrate-restore]
 
-    For details about configuring snapshot repositories on Amazon Web Services (AWS), Google Cloud Storage (GCS), or Azure Blob Storage, check [manage Snapshot Repositories](../deploy-manage/tools/snapshot-and-restore/cloud-enterprise.md).
+Once the repository has been registered and verified, you are ready to restore any data from any of its snapshots to your new cluster. You can do this using {{kib}} management UI, or directly with the {{es}} API.
 
-    If you’re migrating [searchable snapshots](../deploy-manage/tools/snapshot-and-restore/searchable-snapshots.md), the repository name must be identical in the source and     destination clusters.
-    :::
+For extra details about the contents of a snapshot refer to [](/deploy-manage/tools/snapshot-and-restore.md#snapshot-contents).
 
-    ::::
+To start the restore process:
 
-4. Start the Restore process.
+1. Open Kibana and go to **Management** > **Snapshot and Restore**.
+2. Under the **Snapshots** tab, you can find the available snapshots from your newly added snapshot repository. Select any snapshot to view its details, and from there you can choose to restore it.
+3. Select **Restore**.
+4. Select the index or indices you wish to restore.
+5. Optionally, configure additional restore options, such as **Restore aliases**, **Restore global state**, or **Restore feature state**.
 
-    1. Open Kibana and go to **Management** > **Snapshot and Restore**.
-    2. Under the **Snapshots** tab, you can find the available snapshots from your newly added snapshot repository. Select any snapshot to view its details, and from there you can choose to restore it.
-    3. Select **Restore**.
-    4. Select the indices you wish to restore.
-    5. Configure any additional index settings.
-    6. Select **Restore snapshot** to begin the process.
+    Refer to [Restore a snapshot](/deploy-manage/tools/snapshot-and-restore/restore-snapshot.md) for more details about restore operations in {{es}}, including API based examples.
+    
+6. Select **Restore snapshot** to begin the process.
 
-5. Verify that the new index is restored in your deployment with this query:
+7. Verify that the new index is restored in your deployment with this query:
```

Comment:
could round this out to indicate that you could run it for each index you want to check